src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj

```diff
@@ -14,6 +14,9 @@
   <PropertyGroup>
```

Comment:
Please bump the version and update PackageReleaseNotes here

Reply:
Please look at an example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj#L13)